htmd/smallmol/smallmol.py

```diff
@@ -1176,6 +1176,11 @@ class NoConformerError(Exception):
         elif not isinstance(ids, list):
             raise ValueError('The argument ids should be a list of confomer ids')
 
+        # checks if coordintates exist
+        n_coords = self.getCoords().shape[0]
+        if n_coords == 0:
+            raise ValueError('No coordinates found. You need at least one atom coordinate.')
```

Comment:
`No coordinates found. You need at least one atom coordinate to convert SmallMol to a Molecule object.`
make it like this a tiny bit more descriptive.